bb/resources/native-image-tests/run-normtests

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+TIMBRE_LEVEL=':fatal' bin/kaocha --focus :norm "$@"
```

Comment:
I think you don't really need this file, do you? The norm tests should simply be able to be called with 
`bb test norm`